app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -38,7 +38,8 @@ class PatientSummaryUpdate : Update<PatientSummaryModel, PatientSummaryEvent, Pa
       )
```

Comment:
Typo in the commit message? 

"Show diagnosis error **if** diagnosis is not recorded and done is clicked"